dataschema/schema2scala.py

```diff
@@ -313,18 +312,21 @@ def _get_column_annotations(self, column: Schema.Column,
 
     def _get_column_struct_template(self,
                                     column: Schema.Column) -> Optional[str]:
-        if column.info.label == Schema_pb2.ColumnInfo.LABEL_OPTIONAL:
-            return 'Option'
-        if column.info.label == Schema_pb2.ColumnInfo.LABEL_REQUIRED:
-            return None
-        if column.info.label != Schema_pb2.ColumnInfo.LABEL_REPEATED:
-            raise ValueError(
-                f'Label not specified for column `{column.name()}`')
-        if column.info.repeated_semantics == Schema_pb2.ColumnInfo.REPEATED_ARRAY:
-            return 'Array'
-        if column.info.repeated_semantics == Schema_pb2.ColumnInfo.REPEATED_SET:
-            return 'Set'
-        return 'Seq'
+        templates = []
+        if column.info.column_type == Schema_pb2.ColumnInfo.TYPE_ARRAY:
+            templates.append('Array')
+        elif column.info.column_type == Schema_pb2.ColumnInfo.TYPE_SET:
+            templates.append('Set')
+        elif column.is_repeated():
+            if column.info.repeated_semantics == Schema_pb2.ColumnInfo.REPEATED_ARRAY:
+                templates.append('Array')
+            elif column.info.repeated_semantics == Schema_pb2.ColumnInfo.REPEATED_SET:
+                templates.append('Set')
+            else:
+                templates.append('Seq')
+        if column.is_optional():
+            templates.append('Option')
```

Comment:
Does the order in which templates are appended matter? Does Option need to come first?

Reply:
Actually it does e.g. py: Optional[List[int]] => Option[Seq[Long]] vs. List[Optional[int]] => Seq[Option[int]]